client/sites-dashboard-v2/dotcom-style.scss

```diff
@@ -1,7 +1,50 @@
 // Add new Dotcom specific fly-out panel styles to this file
+
+@import "@automattic/typography/styles/variables";
+@import "@wordpress/base-styles/variables";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
 .wpcom-site .a4a-layout-with-columns__container {
 	background: var(--color-sidebar-background);
 }
 .wpcom-site .main.a4a-layout.sites-dashboard.sites-dashboard__layout .sites-overview {
 	background: var(--studio-white);
 }
+
+/*
+	Styles for actions (search, filters)
+*/
+.dataviews-filters__view-actions {
+	align-items: center;
+
+	.components-search-control {
+		@media (min-width: 1014px) {
+			min-width: 390px;
+		}
```

Comment:
Should we use pre-defined breakpoints instead of these?

Reply:
Pushed a fix: https://github.com/Automattic/wp-calypso/pull/89887/commits/81fa7f59a9d46616d171d6cb92b888aab73172c0